components/odh-notebook-controller/config/manager/manager.yaml

```diff
@@ -25,7 +25,8 @@ spec:
           imagePullPolicy: Always
```

Comment:
leaving this at Always here, compared to odh-manifests, since here, you actually work with the :latest tag of odh-notebook controller.